image/registry.Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM registry:2.7
+COPY ./config/registry.yml /etc/docker/registry/config.yml
```

Comment:
This is good for me. But if you want to avoid building an image for the registry, you can use environment variables directives: https://docs.docker.com/registry/configuration/ .